dyn_em/module_initialize_fire.F

```diff
@@ -525,11 +525,14 @@ SUBROUTINE init_domain_rk ( grid &
                 ifms,ifme,jfms,jfme,  &
                 ifts,ifte,jfts,jfte,  &
                 grid%fxlong,grid%fxlat          )
-  call set_ideal_coord( grid%dx,grid%dy, &
-                ids,ide,jds,jde,  &
-                ims,ime,jms,jme,  &
-                its,ite,jts,jte,  &
-                grid%xlong,grid%xlat          )
+! Avoid setting atmospheric coordinates when using nested domains ME
```

Comment:
It seems better to leave the original logic for which domains to use, and fix the assignment of the lat/lon values.